Docker image build error

### Summary

During our "free disk space" step, we run into an error when purging packages:

```
Run sudo docker rmi $(docker image ls -aq) >/dev/null 2>&1 || true
  sudo docker rmi $(docker image ls -aq) >/dev/null 2>&1 || true
  sudo rm -rf \
    /usr/share/dotnet /usr/local/lib/android /opt/ghc \
    /usr/local/share/powershell /usr/share/swift /usr/local/.ghcup \
    /usr/lib/jvm || true
  echo "some directories deleted"
  sudo apt install aptitude -y >/dev/null 2>&1
  sudo aptitude purge aria2 ansible azure-cli shellcheck rpm xorriso zsync \
    esl-erlang firefox gfortran google-chrome-stable \
    google-cloud-sdk imagemagick \
    libmagickcore-dev libmagickwand-dev libmagic-dev ant ant-optional kubectl \
    mercurial apt-transport-https mono-complete libmysqlclient \
    unixodbc-dev yarn chrpath libssl-dev libxft-dev \
    libfreetype6 libfreetype6-dev libfontconfig1 libfontconfig1-dev \
    snmp pollinate libpq-dev postgresql-client powershell ruby-full \
    sphinxsearch subversion mongodb-org azure-cli microsoft-edge-stable \
    temurin-11-jdk temurin-8-jdk temurin-17-jdk temurin-21-jdk \
    llvm-16-dev llvm-17-dev llvm-18-dev \
    -y -f >/dev/null 2>&1
  sudo aptitude purge google-cloud-sdk -f -y >/dev/null 2>&1
  sudo aptitude purge microsoft-edge-stable -f -y >/dev/null 2>&1 || true
  sudo apt purge microsoft-edge-stable -f -y >/dev/null 2>&1 || true
  sudo aptitude purge '~n ^mysql' -f -y >/dev/null 2>&1
  sudo aptitude purge '~n ^php' -f -y >/dev/null 2>&1
  sudo aptitude purge '~n ^dotnet' -f -y >/dev/null 2>&1
  sudo apt-get autoremove -y >/dev/null 2>&1
  sudo apt-get autoclean -y >/dev/null 2>&1
  echo "some packages purged"
  shell: /usr/bin/bash -e {0}
some directories deleted
Error: Process completed with exit code 100.
```

### Platform

Ubuntu 24.04.3

### Version

0.8.7

### Python version

_No response_

--- a/.github/workflows/docker-jupyter-image.yml
+++ b/.github/workflows/docker-jupyter-image.yml
@@ -36,15 +36,15 @@ jobs:
             sphinxsearch subversion mongodb-org azure-cli microsoft-edge-stable \
             temurin-11-jdk temurin-8-jdk temurin-17-jdk temurin-21-jdk \
             llvm-16-dev llvm-17-dev llvm-18-dev \
-            -y -f >/dev/null 2>&1
-          sudo aptitude purge google-cloud-sdk -f -y >/dev/null 2>&1
+            -y -f >/dev/null 2>&1 || true
+          sudo aptitude purge google-cloud-sdk -f -y >/dev/null 2>&1 || true
           sudo aptitude purge microsoft-edge-stable -f -y >/dev/null 2>&1 || true
           sudo apt purge microsoft-edge-stable -f -y >/dev/null 2>&1 || true
-          sudo aptitude purge '~n ^mysql' -f -y >/dev/null 2>&1
-          sudo aptitude purge '~n ^php' -f -y >/dev/null 2>&1
-          sudo aptitude purge '~n ^dotnet' -f -y >/dev/null 2>&1
-          sudo apt-get autoremove -y >/dev/null 2>&1
-          sudo apt-get autoclean -y >/dev/null 2>&1
+          sudo aptitude purge '~n ^mysql' -f -y >/dev/null 2>&1 || true
+          sudo aptitude purge '~n ^php' -f -y >/dev/null 2>&1 || true
+          sudo aptitude purge '~n ^dotnet' -f -y >/dev/null 2>&1 || true
+          sudo apt-get autoremove -y >/dev/null 2>&1 || true
+          sudo apt-get autoclean -y >/dev/null 2>&1 || true
           echo "some packages purged"
 
       - name: Check disk space
@@ -106,15 +106,15 @@ jobs:
             sphinxsearch subversion mongodb-org azure-cli microsoft-edge-stable \
             temurin-11-jdk temurin-8-jdk temurin-17-jdk temurin-21-jdk \
             llvm-16-dev llvm-17-dev llvm-18-dev \
-            -y -f >/dev/null 2>&1
-          sudo aptitude purge google-cloud-sdk -f -y >/dev/null 2>&1
+            -y -f >/dev/null 2>&1 || true
+          sudo aptitude purge google-cloud-sdk -f -y >/dev/null 2>&1 || true
           sudo aptitude purge microsoft-edge-stable -f -y >/dev/null 2>&1 || true
           sudo apt purge microsoft-edge-stable -f -y >/dev/null 2>&1 || true
-          sudo aptitude purge '~n ^mysql' -f -y >/dev/null 2>&1
-          sudo aptitude purge '~n ^php' -f -y >/dev/null 2>&1
-          sudo aptitude purge '~n ^dotnet' -f -y >/dev/null 2>&1
-          sudo apt-get autoremove -y >/dev/null 2>&1
-          sudo apt-get autoclean -y >/dev/null 2>&1
+          sudo aptitude purge '~n ^mysql' -f -y >/dev/null 2>&1 || true
+          sudo aptitude purge '~n ^php' -f -y >/dev/null 2>&1 || true
+          sudo aptitude purge '~n ^dotnet' -f -y >/dev/null 2>&1 || true
+          sudo apt-get autoremove -y >/dev/null 2>&1 || true
+          sudo apt-get autoclean -y >/dev/null 2>&1 || true
           echo "some packages purged"
 
       - name: Check disk space

--- a/.github/workflows/docker-marimo-image.yml
+++ b/.github/workflows/docker-marimo-image.yml
@@ -36,15 +36,15 @@ jobs:
             sphinxsearch subversion mongodb-org azure-cli microsoft-edge-stable \
             temurin-11-jdk temurin-8-jdk temurin-17-jdk temurin-21-jdk \
             llvm-16-dev llvm-17-dev llvm-18-dev \
-            -y -f >/dev/null 2>&1
-          sudo aptitude purge google-cloud-sdk -f -y >/dev/null 2>&1
+            -y -f >/dev/null 2>&1 || true
+          sudo aptitude purge google-cloud-sdk -f -y >/dev/null 2>&1 || true
           sudo aptitude purge microsoft-edge-stable -f -y >/dev/null 2>&1 || true
           sudo apt purge microsoft-edge-stable -f -y >/dev/null 2>&1 || true
-          sudo aptitude purge '~n ^mysql' -f -y >/dev/null 2>&1
-          sudo aptitude purge '~n ^php' -f -y >/dev/null 2>&1
-          sudo aptitude purge '~n ^dotnet' -f -y >/dev/null 2>&1
-          sudo apt-get autoremove -y >/dev/null 2>&1
-          sudo apt-get autoclean -y >/dev/null 2>&1
+          sudo aptitude purge '~n ^mysql' -f -y >/dev/null 2>&1 || true
+          sudo aptitude purge '~n ^php' -f -y >/dev/null 2>&1 || true
+          sudo aptitude purge '~n ^dotnet' -f -y >/dev/null 2>&1 || true
+          sudo apt-get autoremove -y >/dev/null 2>&1 || true
+          sudo apt-get autoclean -y >/dev/null 2>&1 || true
           echo "some packages purged"
 
       - name: Check disk space
@@ -106,15 +106,15 @@ jobs:
             sphinxsearch subversion mongodb-org azure-cli microsoft-edge-stable \
             temurin-11-jdk temurin-8-jdk temurin-17-jdk temurin-21-jdk \
             llvm-16-dev llvm-17-dev llvm-18-dev \
-            -y -f >/dev/null 2>&1
-          sudo aptitude purge google-cloud-sdk -f -y >/dev/null 2>&1
+            -y -f >/dev/null 2>&1 || true
+          sudo aptitude purge google-cloud-sdk -f -y >/dev/null 2>&1 || true
           sudo aptitude purge microsoft-edge-stable -f -y >/dev/null 2>&1 || true
           sudo apt purge microsoft-edge-stable -f -y >/dev/null 2>&1 || true
-          sudo aptitude purge '~n ^mysql' -f -y >/dev/null 2>&1
-          sudo aptitude purge '~n ^php' -f -y >/dev/null 2>&1
-          sudo aptitude purge '~n ^dotnet' -f -y >/dev/null 2>&1
-          sudo apt-get autoremove -y >/dev/null 2>&1
-          sudo apt-get autoclean -y >/dev/null 2>&1
+          sudo aptitude purge '~n ^mysql' -f -y >/dev/null 2>&1 || true
+          sudo aptitude purge '~n ^php' -f -y >/dev/null 2>&1 || true
+          sudo aptitude purge '~n ^dotnet' -f -y >/dev/null 2>&1 || true
+          sudo apt-get autoremove -y >/dev/null 2>&1 || true
+          sudo apt-get autoclean -y >/dev/null 2>&1 || true
           echo "some packages purged"
 
       - name: Check disk space

